html/sqlvalidator/viewsqlquery.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>SQL Query Viewer Index page</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="description" content="Elements Chrome Extension">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css"
+          integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css"
+          integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
+    <link rel="stylesheet" href="../../css/pureovewrite.css">
+    <script src="../../js/sqlvalidator/render.js"></script>
+    <script src="../../js/sqlvalidator/sqlvalidator.js"></script>
+</head>
+
+<body>
+<div class="header">
+    <h1 class="title with-navigation">
+        <a id="indexbutton" href="./sqlvalidator.html" class="pure-button title-nav-button">
+            <i class="fa fa-angle-left fa-sm"></i>
+            <span>Back</span>
+        </a>
+        SQL Query Viewer
+    </h1>
+</div>
+
+<div class="pure-form pure-form-stacked">
+    <button class="button-primary button-large pure-button" id="formatsqlstatement" type="button">Show the Real Stuff</button>
+    <div id="sqlquery-input">
+        <div class="input-textfield pure-u-1">
+            <div class="toolbar align-items-right pure-button-group" role="group">
+                <p><b>Input</b></p>
+            </div>
+            <textarea id="inputSql-stmt" rows="5" cols="50" class="input-textfield"></textarea>
+        </div>
+    </div>
+    <div id="sqlquery-output">
+        <table class="pure-table" id="metadataTable">
```

Comment:
The css on this guy is being a bit funny, it's a little wider than the actual view width